sentry_zendesk/__init__.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+from django.conf import settings  # noqa
+import logging
+
+
+try:
+    VERSION = __import__('pkg_resources') \
```

Comment:
Yeah, I agree it is better. I was just following how the official plugins do. I'll change that.